src/components/settings/EditProfile.tsx

```diff
@@ -17,38 +18,43 @@ type EditProfileProps = {
   preferredName: string;
   email: string;
   bio: string;
+  dob: string;
+  university: string;
+  // industries: string[];
   editMode: boolean;
   toggleEditMode: Dispatch<SetStateAction<boolean>>;
 };
 
 // Define the Zod schema for form validation
-const formSchema = z.object({
-  name: z.string().min(1, "Full name is required"),
-  emailAddress: z.string().email("Invalid email address"),
-  dateOfBirth: z.string().min(1, "Date of Birth is required"),
-  biography: z.string().min(1, "Bio is required"),
-  selectedUniversity: z.string().min(1, "College/University is required"),
-  topIndustries: z
-    .array(z.string())
-    .min(1, "At least one industry must be selected"),
-});
+const formSchema = z
+  .object({
+    name: z.string().min(1, "Full name is required"),
+    emailAddress: z.string().email("Invalid email address"),
+    dateOfBirth: z.string().min(1, "Date of Birth is required"),
+    biography: z.string().min(1, "Bio is required"),
+    selectedUniversity: z.string().min(1, "College/University is required"),
+    topIndustries: z
+      .array(z.string())
+      .min(1, "At least one industry must be selected"),
+  })
+  .partial();
 
 const EditProfile = ({
   preferredName,
   email,
   bio,
-  // dateOfBirth,
-  // selectedUniversity,
-  // topIndustries,
+  dob,
+  university,
+  // industries,
   editMode,
   toggleEditMode,
 }: EditProfileProps) => {
   const [name, setName] = useState(preferredName);
   const [emailAddress, setEmailAddress] = useState(email);
-  const [dateOfBirth, setDateOfBirth] = useState("");
+  const [dateOfBirth, setDateOfBirth] = useState(dob);
   const [biography, setBiography] = useState(bio);
-  const [selectedUniversity, setSelectedUniversity] = useState("");
-  const [topIndustries, setTopIndustries] = useState<string[]>([]);
+  const [selectedUniversity, setSelectedUniversity] = useState(university);
+  const [topIndustries, setTopIndustries] = useState<string[]>([]); //Set to empty array always but will need to figure out how to set initial value
```

Comment:
We can figure this out, does putting industries not work here?

Reply:
I tried that at first, it displayed the current industries, but didn't let me select or change it after that.